### DISCUSSION TITLE
Can't see which reusable workflow is called in the logs

### DISCUSSION THREAD

**Discussion by sgandon** [community/community#65554] [Actions]
### Select Topic Area

Product Feedback

### Body

Hello,
When calling reusable workflows there is not way to see a link to the workflow that is actually called and doing the work.
I have tried to create a step in my reused workflow in order to show it's location
```
      - name: Print this workflow url
        run: |
          WORKFLOW_REF=${{ github.workflow_ref }}
          REPO=$(echo $WORKFLOW_REF | cut -d'/' -f1,2)
          FILE_PATH=$(echo $WORKFLOW_REF | cut -d'/' -f3- | cut -d'@' -f1)
          BRANCH=$(echo $WORKFLOW_REF | cut -d'@' -f2 | cut -d'/' -f3-)
          echo "https://github.com/$REPO/blob/$BRANCH/$FILE_PATH"
```
But is it miserably failing as the `github.workflow_ref` is referencing the inital calling workflow and not the current one.
Is there a way to achieve this or can this be an enhancement ?

**Comment by nirav-gajera** [comment]
try:
```

- name: Print this workflow url
        run: |
          WORKFLOW_URL=${{ github.event.workflow_url }}
          echo "https://github.com/$WORKFLOW_URL"
```

**Reply by sgandon** [reply]
thank for this answer but the event.workflow_url is the workflow that initiated the event and not the one that is handling it.
